.vib/wordpress/cypress/cypress/integration/wordpress_spec.js

```diff
@@ -2,88 +2,35 @@
 
 import { random } from './utils';
 
-it('contains the sample blogpost and a comment', () => {
+it('allows login/logout', () => {
+  cy.login();
+  cy.get('#login_error').should('not.exist');
+  cy.visit('wp-login.php?loggedout=true&wp_lang=en_US');
+  cy.contains('.message', 'logged out');
+});
+
+it('allows adding a comment to sample blogpost', () => {
   cy.visit('/');
   cy.get('.wp-block-query');
   cy.get('.wp-block-post-title a').click();
-  cy.fixture('helloworld').then((hw) => {
-    cy.contains('.wp-block-post-title', hw.title);
-    cy.contains('.wp-block-post-content', hw.content);
-  });
-
-  cy.get('.wp-block-post-title').click();
-  cy.get('.commentlist').should('have.length', 1);
-  cy.fixture('helloworld').then((hw) => {
-    cy.get('.comment-author').should('contain.text', hw.commenter);
-
-    cy.fixture('helloworld').then((hw) => {
-      cy.get('#comment').type(hw.comment).should('have.value', hw.comment);
-    });
-
-    cy.fixture('user').then((user) => {
-      cy.get('#author').type(user.username).should('have.value', user.username);
-      cy.get('#email').type(user.email).should('have.value', user.email);
-    });
-
-    cy.get('#submit').click();
+  cy.fixture('posts').then((post) => {
+    cy.contains('.wp-block-post-title', post.title);
+    cy.contains('.wp-block-post-content', post.content);
```

Comment:
Hmm it took me some moments to understand this. As this is a test that deals with commenting, does it make sense to assert other entities (_Posts_) here? I'm not against it when there is a need, but I think that the title and content of the post are not relevant to commenting. WDYT?

Reply:
I modified the test to focus on 'comments' and I just used the entry from 'posts' to select the proper post (`Hello world` one. Hope it's better now!